langchain/retrievers/azure_cognitive_search.py

```diff
@@ -33,7 +33,8 @@ class AzureCognitiveSearchRetriever(BaseRetriever):
     """ClientSession, in case we want to reuse connection for better performance."""
     content_key: str = "content"
     """Key in a retrieved result to set as the Document page_content."""
-
+    top_k: int = 3
```

Comment:
Makes sense to make it `Optional` & let default be `None` for backwards compatibility